charts/kafka-operator/templates/crds.yaml

```diff
@@ -3907,6 +3907,21 @@ spec:
                         external listener advertise address according to the description
                         of the "hostnameOverride" field.
                       type: object
+                    nodePortNodeAddressType:
```

Comment:
**Question**: are we sure this should be here?
I think this should only be in your other PR.